app/lib/common/widgets/html_editor/html_editor.dart

```diff
@@ -293,39 +306,52 @@ class HtmlEditorState extends State<HtmlEditor> {
         linkMobileToolbarItem,
         quoteMobileToolbarItem,
       ],
+      toolbarHeight: 48,
       editorState: editorState,
-      child: MobileFloatingToolbar(
-        editorScrollController: editorScrollController,
-        editorState: editorState,
-        toolbarBuilder: (context, anchor, closeToolbar) {
-          return AdaptiveTextSelectionToolbar.editable(
-            clipboardStatus: ClipboardStatus.pasteable,
-            onCopy: () {
-              copyCommand.execute(editorState);
-              closeToolbar();
-            },
-            onCut: () => cutCommand.execute(editorState),
-            onPaste: () => pasteCommand.execute(editorState),
-            onSelectAll: () => selectAllCommand.execute(editorState),
-            onLiveTextInput: null,
-            onLookUp: null,
-            onSearchWeb: null,
-            onShare: null,
-            anchors: TextSelectionToolbarAnchors(primaryAnchor: anchor),
-          );
-        },
-        child: AppFlowyEditor(
-          // widget pass through
-          editable: widget.editable,
-          shrinkWrap: widget.shrinkWrap,
-          autoFocus: widget.autoFocus,
-          header: widget.header,
-          // local states
-          editorState: editorState,
-          editorScrollController: editorScrollController,
-          editorStyle: mobileEditorStyle(),
-          footer: generateFooter(),
-        ),
+      child: Column(
```

Comment:
It's because of the internal layout of Appflowy widget which is using a lot of scrollable widgets. And making it expandable dynamically is really a complex thing. This pattern is something which I took from the package source code itself (from their example code) and it seems like wrapping it in column solves layout issue.